app/models/d_m_channel.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
```

Comment:
is it expected to be d_m_channel.rb and not dm_channel.rb ?

Reply:
Yes this way the naming is compatible with what zeitwerk is expecting. `dm_channel` would translate to `DmChannel` instead.
But this is (hopefully) a temporary model name, at some point we should be able to merge the existing `DirectMessageChannel` model in it (and use the proper name).